Xwt/Xwt.Backends/IToggleCellViewFrontend.cs

```diff
@@ -27,7 +27,7 @@
 namespace Xwt.Backends
 {
 
-	public interface IToggleCellViewFrontend : ICellViewFrontend, IEditableCellViewFrontend
+	public interface IToggleCellViewFrontend : ICellViewFrontend, IEditableCellViewFrontend<CheckBoxState>
```

Comment:
Toggle has no mixed state and should be `IEditableCellViewFrontend<bool>`, probably `IToggleCellViewFrontend` should not be based on `IEditableCellViewFrontend` but only the implementing frontend classes (otherwise it would be ambiguous for check boxes).

Reply:
Fixed